src/components/learner-credit-management/assignment-modal/AssignmentModalContent.jsx

```diff
@@ -207,6 +206,7 @@ AssignmentModalContent.propTypes = {
   courseRun: PropTypes.shape({
     enrollBy: PropTypes.string,
     start: PropTypes.string,
+    contentPrice: PropTypes.string,
```

Comment:
[sanity check] It looks like this may be a number when seeing its value with a `console.log`?

<img width="1134" alt="image" src="https://github.com/user-attachments/assets/e423f9d5-0416-4109-8db5-759d91ded5c3">

Reply:
Since get_content_price from the NormalizedMetadataSerializer is not returning a singular type at the moment (string for EE (entitlements.price) and int for OCM (first_enrollable_paid_seat_price)) there is always going to be some sort of mismatch. Here is an example price from EE example
![Screenshot 2024-09-18 at 10 49 57 AM](https://github.com/user-attachments/assets/321864ec-9ac5-4515-9d17-fd0feb2bb9d0)

Reply:
In this case, then should the prop type account for the multiple data types to remove this confusion?